### ISSUE TITLE
Allow support for empty property names

### ISSUE THREAD

**Issue by btiernay** [coveooss/json-schema-for-humans#148]
Currently if an object property name is `""`, then the library generates an error:
```
Traceback (most recent call last):
  File "/Users/btiernay/.pyenv/versions/3.9.10/bin/generate-schema-doc", line 8, in <module>
    sys.exit(main())
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/json_schema_for_humans/cli.py", line 76, in main
    generate_schemas_doc(schemas_to_render, template_renderer)
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/json_schema_for_humans/generate.py", line 151, in generate_schemas_doc
    rendered_schemas[schema_to_render.schema_file_name] = _generate_schema(
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/json_schema_for_humans/generate.py", line 136, in _generate_schema
    result = schema_to_render.render(template_renderer, loaded_schemas)
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/json_schema_for_humans/schema/schema_to_render.py", line 54, in render
    intermediate_schema = build_intermediate_representation(schema_file, template_renderer.config, loaded_schemas)
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/json_schema_for_humans/schema/intermediate_representation.py", line 42, in build_intermediate_representation
    intermediate_representation = _build_node(
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/json_schema_for_humans/schema/intermediate_representation.py", line 416, in _build_node
    new_html_id += escape_property_name_for_id(new_property_name)
  File "/Users/btiernay/.pyenv/versions/3.9.10/lib/python3.9/site-packages/json_schema_for_humans/jinja_filters.py", line 195, in escape_property_name_for_id
    if not escaped[0].isalpha():
```

Changing to a non-empty string value fixes the issue.

**Comment by dblanchette** [comment]
Fixed in version 0.41.7

**Comment by btiernay** [comment]
Thank you! 🙇
